vendorpromo/api/v1/vouchery/views.py

```diff
@@ -19,8 +17,8 @@ def post(self, request, *args, **kwargs):
         promo_form = PromoForm(request.POST)
         if not promo_form.is_valid():
             raise HttpResponseBadRequest()
-
-        processor = promo_processor()
+        promo = promo_form.save(commit=False)
```

Comment:
this promo variable doesn't seem to be used

Reply:
agree